lib/data/benefit_cap_data.yml

```diff
@@ -1,15 +1,16 @@
 ---
 default:
   weekly_benefit_caps:
-    single:
-      amount: 350
-      description: "Single"
-    couple:
-      amount: 500
-      description: "Living as a couple (with or without children)"
-    parent:
-      amount: 500
-      description: "A lone parent with 1 or more dependent children"
+    national:
+      single:
+        amount: 350
+        description: "Single"
+      couple:
+        amount: 500
+        description: "Living as a couple (with or without children)"
+      parent:
+        amount: 500
+        description: "A lone parent with 1 or more dependent children"
```

Comment:
I find it confusing that only the `default` version of `weekly_benefit_caps` is updated here. I would've expected the `future` version to be updated in this commit, but it seems to have been done in an earlier commit - before the change to `BenefitCapCalculatorConfiguration#weekly_benefit_caps` which uses it.